mantidproject/mantid
# Fix custom interface help links

Description of work.

Updates the interface page names embedded in the interface help buttons to match those in the docs.

**To test:**

* build the code
* build the docs-qthelp target
* try all help buttons - they should point to a valid page on all interfaces

*No issue*

**Release Notes** 
*Does not need to be in the release notes as we broke it in the dev cycle*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
